README.MD

```diff
@@ -36,4 +30,4 @@ SerialProtocol is inspired by [Arduino-EasyTransfer](https://github.com/madsci10
 #### License
 
 Copyright [Victor Hurdugaci](http://victorhurdugaci.com). All rights reserved.  
-Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
\ No newline at end of file
+Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
```

Comment:
Can you please add a new section at the end of this file called "Contributors" and add your name to it?